tests/test_securityhub.py

```diff
@@ -214,6 +214,34 @@ def test_instance(self):
             },
         )
 
+    def test_instance_findings_filter(self):
+        factory = self.replay_flight_data("test_security_hub_instance_findings_filter")
+        policy = self.load_policy(
+            {
+                "name": "ec2-findings-filter",
+                "resource": "ec2",
+                "filters": ["findings"],
+            },
+            config={"account_id": "101010101111"},
+            session_factory=factory,
+        )
+
+        resources = policy.run()
+        self.assertEqual(len(resources), 1)
+
+        client = factory().client("securityhub")
+        findings = client.get_findings(
```

Comment:
this api call isn't nesc for the test, there isn't any mutation side effect from a filter, the result of finding the resource, and asserting a filter added annotation is enough.

Reply:
deleted